.gitignore

```diff
@@ -66,4 +66,5 @@ html_app/index.html
 .vagrant/
 static/
 
-
+*\.soy\.js
+*\.gss\.css
```

Comment:
Can we not just ignore folders called `gen`? or are there things we want to keep in them?